Integrations/integration-Cisco-umbrella.yml

```diff
@@ -31,495 +31,1834 @@ configuration:
   type: 0
   required: true
 script:
-  script: |2+
-
-
-    // This object describe the result of the http request of getDomainSecurity function
-    // each field has Name,Iinfo & ContextKey  - thats just looks scarry
-    var SecurityResultInfo = {
-        dga_score: { Name: 'DGA', Info: 'Domain Generation Algorithm. This score is generated based on the likeliness of the domain ' +
+  script: |-
+    ''' IMPORTS '''
+
+    import sys
+    import requests
+    import json
+    import os
+    import time
+    import re
+    import urllib
+    from urlparse import urlparse
+    from distutils.util import strtobool
+    from datetime import datetime,timedelta
+
+    # Disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    ''' GLOBALS/PARAMS '''
+
+    API_TOKEN = demisto.params()['APIToken']
+    BASE_URL = demisto.params()['baseURL']
+    USE_SSL = True if demisto.params().get('insecure') else False
+    DEFAULT_HEADERS = {
+        'Authorization': 'Bearer {}'.format(API_TOKEN),
+        'Accept': 'application/json'
+    }
+    # remove proxy if not set to true in params
```

Comment:
Why shouldn't we move all the proxy related stuff to CommonServer?

Reply:
@liorblob does not approve of it.

Reply:
@yardensade sure, make me the bad guy